docker/dockerfiles/be/cn_entrypoint.sh

```diff
@@ -185,12 +163,9 @@ if [[ "x$LOG_CONSOLE" == "x1" ]] ; then
     addition_args="--logconsole"
 fi
 $STARROCKS_HOME/bin/start_cn.sh $addition_args
-ret=$?
-if [[ $ret -ne 0 && "x$LOG_CONSOLE" != "x1" ]] ; then
-    nol=50
-    log_stderr "Last $nol lines of cn.INFO ..."
-    tail -n $nol $STARROCKS_HOME/log/cn.INFO
-    log_stderr "Last $nol lines of cn.out ..."
-    tail -n $nol $STARROCKS_HOME/log/cn.out
-fi
-exit $ret
+
+# The reason why we need to sleep here is to avoid the pod being killed by k8s
+# before the preStop hook is exited.
+# If the CN subprocess fails to start, we also want the entrypoint script to exit as soon as possible.
+# Taking all the above into consideration, we set it to five seconds.
+sleep 5
```

Comment:
this will override the exit code, no matter it is a normal exit or a crash or whatever, better keep the sam return code from start_cn.sh